persistence-jdbc/core/src/main/resources/reference.conf

```diff
@@ -2,8 +2,9 @@
 # Defaults to use for each Akka persistence plugin
 jdbc-defaults.slick {
 
-  # The driver to use
-  driver = "slick.jdbc.H2Profile$"
+  # The Slick profile to use
+  # set to one of: slick.jdbc.PostgresProfile$, slick.jdbc.MySQLProfile$, slick.jdbc.OracleProfile$ or slick.jdbc.H2Profile$
+  # profile = "slick.jdbc.PostgresProfile$"
```

Comment:
We don't define a default profile anymore. As such we don't have the risk of overriding users old and deprecated `slick.driver`.